.changeset/tender-lies-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+Adding the lib/ folder to the exports in package.json
```

Comment:
~~~suggestion
Added the `lib/` directory to the exports in `package.json`. Users can now do things like import controllers into their own projects:

```js
import { ScreenSizeController } from '@rhds/elements/lib/ScreenSizeController.js';
```
~~~